tests/unit/serve_test.ts

```diff
@@ -12,15 +12,14 @@ import {
   assertThrows,
   curlRequest,
   curlRequestWithStdErr,
-  DENO_FUTURE,
   execCode,
   execCode3,
   fail,
   tmpUnixSocketPath,
 } from "./test_util.ts";
 
 // Since these tests may run in parallel, ensure this port is unique to this file
-const servePort = DENO_FUTURE ? 4511 : 4502;
```

Comment:
This should actually use `4502` :D

Reply:
You resolved this conversation. Change to 4502 or keep as 4511?